corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -269,6 +269,17 @@ def get_deleted_form_ids_for_user(domain, user_id):
         )
 
     @staticmethod
+    def get_form_ids_in_domain_by_type(domain, type_):
+        state = doc_type_to_state[type_]
+        with get_cursor(XFormInstanceSQL) as cursor:
+            cursor.execute(
+                'SELECT form_id from get_form_ids_in_domain_by_type(%s, %s)',
+                [domain, state]
+            )
+            results = fetchall_as_namedtuple(cursor)
+            return [result.form_id for result in results]
```

Comment:
Yeah that seems good. Might be a larger change though (looks like lots of dbaccessors are using it)